VideoLocker/build.gradle

```diff
@@ -155,7 +155,7 @@ android {
     }
 
     compileSdkVersion 23
-    buildToolsVersion BUILD_TOOLS_VERSION
+    buildToolsVersion '23.0.3'
```

Comment:
Naively seems weird to be replacing this variable with a particular constant? Is that constant no longer correct?